documentation/operators/src/legal/landing-pages.md

```diff
@@ -10,35 +10,54 @@ Exit Gateway landing page is a great and transparent way to prevent possible tro
 
 ## Avril 14th Exit Gateways
 
-The following snippet it on the 'assets' path under '[http]' section of your config file
+The following snippet needs to modified as described below according to the public identity that you may want to show on this public notice, i.e. your graphics and your email.
+It would allow you to serve it as a landing page resembling the one proposed by [Tor](https://gitlab.torproject.org/tpo/core/tor/-/raw/HEAD/contrib/operator-tools/tor-exit-notice.html) but with all the changes needed to adhere to the Nym's operators case.
+
+[Avril 14th](https://avril14th.org/) is a team of professionals and optimistic realists eager to contribute to a future of privacy, decentralization and connectivity. 
+
+When done with the customization needs as described below, you'll need to upload the file and reference it as the 'landing_page_assets_path' on the '[http]' section of the config.toml file:
```

Comment:
Here use \` instead of \' for landing_page_assets_path and for [http].
so they come out like this: 

... you'll need to upload the file and reference it as the `landing_page_assets_path` on the `[http]`

Reply:
not sure about what you mean, the final sentence would be 
... you'll need to upload the file and reference it as the landing_page_assets_path on the [http]
or 
... you'll need to upload the file and reference it as the 'landing_page_assets_path' on the '[http]' section of the config.toml file

I mean that maybe newcomers don't know which file to edit in order to find and change such [http] entry.
BTW that should be pretty far from real since if you're running a gateway you'd probably already gained some experience with the node configuration..

Reply:
sorry. My bad. I meant to use the code ticks `  \` ` and not quotation ` ' ` ticks to make these two coming out like this:
`landing_page_assets`
`[http]`

Reply:
It was just a syntax suggestion